README.md

```diff
@@ -148,24 +148,24 @@ ORDER BY
 
 |Outlier Type           |Percentage|
 |-----------------------|---------:|
-|PTAX-203 flag (Low)    |35.4%     |
-|Non-person sale (low)  |18.07%    |
-|Non-person sale (high) |10.27%    |
-|High price (raw)       |6.67%     |
-|Anomaly (high)         |6.43%     |
-|Low price (raw)        |4.55%     |
-|Low price (raw & sqft) |4.09%     |
-|PTAX-203 flag (High)   |3.07%     |
-|Home flip sale (high)  |2.26%     |
-|High price (sqft)      |2.03%     |
-|Low price (sqft)       |1.99%     |
-|Anomaly (low)          |1.65%     |
-|High price (raw & sqft)|1.63%     |
-|Home flip sale (low)   |1.15%     |
-|Family sale (low)      |0.65%     |
+|PTAX-203 flag (Low)    |35.5%     |
```

Comment:
**[Nitpick, non-blocking]** One small readability suggestion for this table: If we were to use two decimal places for every percentage instead of truncating the multiples of 10 to one decimal place, then the order of magnitude of each percentage would be easy to gauge just by giving the table a quick scan since each percentage's decimal point would align vertically.

Not a necessary change, since I know the truncation pattern predated this PR, but it's an easy way we could improve the readability of the table if that interests you!

```suggestion
|PTAX-203 flag (Low)    |35.50%     |
```

Reply:
Good eye!